sockeye/inference.py

```diff
@@ -896,6 +900,8 @@ def __init__(self,
                  strip_unknown_words: bool = False) -> None:
         self.context = context
         self.length_penalty = length_penalty
+        self.beam_prune = beam_prune
```

Comment:
can you add a `check_conditition` to make sure that `batch_size` is 1?

Reply:
It doesn't need to be size 1 --- this works with batching.

Reply:
sorry, I meant for the `beam_search_stop`. That works only for batch_size 1, correct?

Reply:
ah, of course, fixed.